src/MSBuild/TerminalLogger/TerminalLogger.cs

```diff
@@ -400,166 +417,170 @@ private void ProjectFinished(object sender, ProjectFinishedEventArgs e)
             UpdateNodeStatus(buildEventContext, null);
         }
 
-        ProjectContext c = new(buildEventContext);
-
-        if (_projects.TryGetValue(c, out Project? project))
+        // Add project summary to the static part of the Console only if verbosity is higher than Quiet.
+        if (Verbosity > LoggerVerbosity.Quiet)
```

Comment:
Can we actually just shortcut quit the mothod for quiet?
It would get the diff more readable

Reply:
Or group the condition with the existing one

Reply:
I agree, shortcut quit is better here.